modules/fundamental/src/package/model/details/qualified_package.rs

```diff
@@ -44,6 +54,36 @@ impl QualifiedPackageDetails {
                 .ok_or(Error::Data("underlying package missing".to_string()))?
         };
 
+        /*
+        let statuses = package_status::Entity::find()
+            .left_join( version_range::Entity )
+            .all(tx)
+            .await?;
+
+        println!("status {:#?}", statuses);
+
+         */
+
+        let statuses = package_status::Entity::find()
+            .columns([
+                version_range::Column::Id,
+                version_range::Column::LowVersion,
+                version_range::Column::LowInclusive,
+                version_range::Column::HighVersion,
+                version_range::Column::HighInclusive,
+            ])
+            .left_join(package::Entity)
+            .join(JoinType::LeftJoin, package::Relation::PackageVersions.def())
+            .left_join(version_range::Entity)
+            .filter(package_status::Column::PackageId.eq(package.id))
+            .filter(SimpleExpr::FunctionCall(
+                Func::cust("version_matches".into_identity().into_iden())
```

Comment:
It's possible to create `Iden`s which can then be re-used. This prevents spreading identifiers throughout the code: https://github.com/trustification/trustify/blob/1d00670178f628f3bd1b803886d03e68a1de44ad/common/src/db/func.rs#L4-L14

Reply:
Good point. Will do.